command.go

```diff
@@ -411,6 +411,102 @@ func (cmd *StringCmd) Float64() (float64, error) {
 	return strconv.ParseFloat(cmd.val, 64)
 }
 
+func (cmd *StringCmd) Scan(vv interface{}) error {
```

Comment:
Don't you want to declare `(err error)`, would simplify the code below, no need for declarations, returns, etc